frontend/src/routes/Governance/components/DiffModal.tsx

```diff
@@ -0,0 +1,47 @@
+/* Copyright Contributors to the Open Cluster Management project */
+
+import { Button, Modal, ModalVariant } from '@patternfly/react-core'
+import { useState } from 'react'
+import { useTranslation } from '../../../lib/acm-i18next'
+import Highlight from 'react-highlight'
+import './github.min.css'
+
+export function DiffModal({
+  diff,
+  kind,
+  name,
+  namespace,
+}: Readonly<{ name: string; diff: string; namespace: string; kind: string }>) {
+  const [isModalOpen, setIsModalOpen] = useState(false)
+  const handleModalToggle = () => {
+    setIsModalOpen((prevIsModalOpen) => !prevIsModalOpen)
+  }
+  const { t } = useTranslation()
+  return (
+    <>
+      {diff && (
+        <>
+          {'-'}
```

Comment:
Should this have spaces between the hyphen?
```suggestion
          {' - '}
```